conda/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+python:
+  - 3.7
```

Comment:
I'm not sure we need this file if we always pass the version to the `conda build` command?

Reply:
So i checked locally and it does override the version, so i think you can remove that file.
And maybe you can also remove `python` from the `conda/meta.yaml` file?

Reply:
"maybe"? Can I or not?

Reply:
"Maybe" here meant "can you give it a go and we can see if it still works as we think"?